README.md

```diff
@@ -123,7 +124,7 @@ steps:
     BRIDGE_COVERITY_CONNECT_USER_NAME: $(COVERITY_USER)
     BRIDGE_COVERITY_CONNECT_USER_PASSWORD: $(COVERITY_PASSPHRASE)
```

Comment:
in all 3 workflow examples few places we are having '' on right side of variable value placeholder rest don't can we make it consistent?

Reply:
removed